examples/AspNet6/Program.cs

```diff
@@ -0,0 +1,89 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System.Reflection;
+using Examples.AspNet6;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+
+builder.Services.AddSwaggerGen();
+
+// OpenTelemetry
+var assemblyVersion = Assembly.GetExecutingAssembly().GetName().Version?.ToString() ?? "unknown";
+
+var resourceBuilder = ResourceBuilder.CreateDefault()
+        .AddService(Tracing.ActivitySourceName, serviceVersion: assemblyVersion, serviceInstanceId: Environment.MachineName)
+        .AddTelemetrySdk();
+
+builder.Logging.ClearProviders();
+
+builder.Logging.AddConsole();
+
+builder.Logging.AddOpenTelemetry(options =>
+{
+    options.SetResourceBuilder(resourceBuilder)
+        .AddConsoleExporter()
+        .AddOtlpExporter();
+});
+
+builder.Services.AddOpenTelemetryTracing(options =>
+{
+    options
+       .SetResourceBuilder(resourceBuilder)
+       .SetSampler(new AlwaysOnSampler())
+       .AddHttpClientInstrumentation()
+       .AddSource(Tracing.ActivitySourceName)
+       .AddConsoleExporter()
```

Comment:
the existing example select exporter based on user configuration, we'll need to keep the similar behavior.